File: a3.cpp
// Basic Calculator
#include<iostream>
using namespace std;
int main()
{
    int a, b, sum, sub, multi; 
    float div;
    cout<<"Enter First No.: ";
    cin>>a;
    cout<<"Enter Second No.: ";
    cin>>b;

    sum=a+b;
    sub=a-b;
    multi=a*b;
    div=a/b;

    cout<<"The Sum is "<<sum<<endl;
    cout<<"The Sub is "<<sub<<endl;
    cout<<"The Multi is "<<multi<<endl;
    cout<<"The Div is "<<div<<endl;
    return 0;
}

File: a3.java
// Basic Calculator
import java.util.Scanner;
public class a3{
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        float a, b, add, sub, multi;
        float div;
        System.out.println("Enter first no. : ");
        a = sc.nextFloat();
        System.out.println("Enter Second no. : ");
        b = sc.nextFloat();
        add=a+b;
        sub=a-b;
        multi=a*b;
        div=(a/b);
        
        System.out.println("Addition is : "+add);
        System.out.println("Subtraction is : "+sub);
        System.out.println("Multiplication is : "+multi);
        System.out.println("Division is : "+div);

    }
}